sdk/aqueduct/integrations/connect_config.py

```diff
@@ -0,0 +1,150 @@
+from enum import Enum
+from typing import Dict, Optional, Union
+
+from aqueduct.constants.enums import MetaEnum, ServiceType
+from aqueduct.error import InternalAqueductError
+from pydantic import BaseModel, Extra, Field
+
+"""Copied mostly over from `aqueduct_executor/operators/connectors/data/config.py` for now, please keep them in sync."""
+
+
+class BaseConnectionConfig(BaseModel):
```

Comment:
What is the end goal for these structs? Should we move them to a `shared` directory in the sdk and have `aqueduct_executor` refer to these?

Reply:
Yeah I would like to get it shared between them the same way our serialization logic is. But there are a couple discrepancies that were introduced to get this working on the SDK side, so to keep this well-scoped, I figured we could duplicate them for now to get the feature out and have a cleanup task for merging them.

Reply:
https://linear.app/aqueducthq/issue/ENG-2235/share-integrationconfig-between-sdk-and-executor